docs/howto/advanced-options.rst

```diff
@@ -1 +1,5 @@
+:orphan:
+
+.. _advanced-options:
+
```

Comment:
Are these changes the result of a tool ?

Why is this file considered `orphan` (and what does that do ?), but not `howto/configure.rst` ?

Reply:
`howto/configure.rst` is directly referenced in a toctree (see `howto_index.rst`), along with `howto/file_naming`. All files in the `howto` that are not listed in that toctree, but still used in documentation links (`advanced-options.rst` and `troubleshoot.rst`) are orphan. They raise a warning (treaded as error by the linter) if the `:orphan:` tag is not specified at the top of the file.

Reply:
Ok! Thank you for the explanations :)